server/src/test/scala/manifests/ManifestProviderSpec.scala

```diff
@@ -4,36 +4,38 @@ import akka.stream.Materializer
 import akka.stream.scaladsl.Sink
 import akka.testkit.TestProbe
 import controllers.ArrivalGenerator
-import manifests.passengers.{BestAvailableManifest, HistoricManifestPax}
+import manifests.passengers.{BestAvailableManifest, ManifestPaxCount}
 import services.crunch.VoyageManifestGenerator.euPassport
 import services.crunch.deskrecs.DynamicRunnableDeskRecs.HistoricManifestsProvider
 import services.crunch.deskrecs.OptimisationProviders
 import services.crunch.{CrunchTestLike, VoyageManifestGenerator}
 import uk.gov.homeoffice.drt.arrivals.{Arrival, VoyageNumber}
 import uk.gov.homeoffice.drt.ports.PortCode
+import uk.gov.homeoffice.drt.ports.SplitRatiosNs.SplitSources
 import uk.gov.homeoffice.drt.time.SDateLike
 
 import scala.concurrent.Future
 
-case class MockManifestLookupService(bestAvailableManifest: BestAvailableManifest,historicManifestPax:HistoricManifestPax)
+case class MockManifestLookupService(bestAvailableManifest: BestAvailableManifest, historicManifestPax: ManifestPaxCount)
                                     (implicit mat: Materializer) extends ManifestLookupLike {
   override def maybeBestAvailableManifest(arrivalPort: PortCode,
                                           departurePort: PortCode,
                                           voyageNumber: VoyageNumber,
                                           scheduled: SDateLike): Future[(UniqueArrivalKey, Option[BestAvailableManifest])] =
     Future.successful((UniqueArrivalKey(arrivalPort, departurePort, voyageNumber, scheduled), Option(bestAvailableManifest)))
 
-  override def historicManifestPax(arrivalPort: PortCode, departurePort: PortCode, voyageNumber: VoyageNumber, scheduled: SDateLike): Future[(UniqueArrivalKey, Option[HistoricManifestPax])] =
+  override def historicManifestPax(arrivalPort: PortCode, departurePort: PortCode, voyageNumber: VoyageNumber, scheduled: SDateLike): Future[(UniqueArrivalKey, Option[ManifestPaxCount])] =
     Future.successful((UniqueArrivalKey(arrivalPort, departurePort, voyageNumber, scheduled), Option(historicManifestPax)))
 
 }
 
 class ManifestProviderSpec extends CrunchTestLike {
   val arrival: Arrival = ArrivalGenerator.arrival(iata = "BA0001", schDt = "2021-06-01T12:00Z")
   val manifestForArrival: BestAvailableManifest = BestAvailableManifest(VoyageManifestGenerator.manifestForArrival(arrival, List(euPassport)))
-  val manifestHistoricForArrival: HistoricManifestPax = HistoricManifestPax(VoyageManifestGenerator.manifestForArrival(arrival, List(euPassport)))
+  val manifestHistoricForArrival: ManifestPaxCount = ManifestPaxCount(VoyageManifestGenerator.manifestForArrival(arrival, List(euPassport)),
+    SplitSources.ApiSplitsWithHistoricalEGateAndFTPercentages)
```

Comment:
Should the source be `Historic` if the variable's name is right (`manifestHistoricForArrival`)?